tests/test_client.py

```diff
@@ -1921,6 +1936,20 @@ def __contains__(self, name):
             http.requests[0]['body'])['code'][0]
         self.assertEqual(code, request_code)
 
+    def test_exchange_with_pkce(self):
+        http = http_mock.HttpMockSequence([
+            ({'status': http_client.OK}, b'access_token=SlAV32hkKG'),
+        ])
+        flow = client.OAuth2WebServerFlow('client_id+1', scope='foo',
+                                          redirect_uri='http://example.com',
+                                          pkce=True)
+        flow.code_verifier = b'__TEST_VERIFIER__'
+        credentials = flow.step2_exchange(code='some random code', http=http)
```

Comment:
It's already validated in other tests, and nothing in this changeset will modify that.

I'll change it to not assign the result of `step2_exchange()` to a variable.